### ISSUE TITLE
some NPCs "lagging/stuttering" during dialogues scenes 

### ISSUE THREAD

**Issue by vraixen** [34736384/genshin-fps-unlock#71]
When npcs walk to reach to conversation sometimes they start "fizzing" or lagging... i don't know how to explain it in english but i think is related to the game engine

**Comment by ManManGuy10** [comment]
Yeah I have this too, i'm pretty sure everyone who uses this does. I imagine it has something to do with the engine or just the way they've animated cutscenes/dialogs. Its not game breaking, so its not a major issue, I doubt there is anything that can be done about it on this end.

**Comment by xymeng16** [comment]
me too, something like shaking happens to some NPCs. But I don't think it's a huge problem, lol.

**Comment by Sleepydragn1** [comment]
For those Googling from the future, like I was:

Capping your frames to 120 FPS using the unlock tool itself appears to prevent the NPC stuttering. I assume this works since the iOS version can also go up to 120 FPS. I still occasionally see NPCs in dialogues that end a walk animation and then appear to stutter a tiny bit back, almost like they're about to start continually stuttering, but it doesn't happen.

Otherwise, I did try matching the frame limit to my refresh rate (170), but that still causes the stutter. Additionally, capping FPS via other methods (in my case I tried using MSI Afterburner with RivaTuner) doesn't seem to resolve the issue, which is unfortunate since I'd love to have a hotkey to toggle the cap on before dialogues.

That being said, obviously it's up to you if limiting your FPS to 120 is worth avoiding this issue. I'm a little unsure myself if I'll keep the cap.
